index.src.html

```diff
@@ -1172,7 +1218,7 @@ <h4 id="algorithm-prevent-silent-access" algorithm>Prevent Silent Access</h4>
 
     :   {{Credential/[[type]]}}
     ::  The {{PasswordCredential}} [=interface object=] has an internal slot named `[[type]]` whose
-        value is "<dfn const for="Credential/[[type]]">`password`</dfn>".
+        value is "<code><dfn const for="Credential/[[type]]">password</dfn></code>".
```

Comment:
Why change from ``` `...` ``` to `<code>...</code>` here? It seems to run counter to

Reply:
Not sure what I was thinking there, please ignore.